examples/LexActivator.pas

```diff
@@ -499,14 +603,133 @@ function GetLicenseAllowedActivations: LongWord;
+
+    RESULT: Allowed deactivations of the license.
+
+    EXCEPTIONS: ELAFailException, ELAProductIdException, ELATimeException,
+    ELATimeModifiedException
+*)
+
+function GetLicenseAllowedDeactivations: LongWord;
+
+(*
+    FUNCTION: GetLicenseCreationDate()
+
+    PURPOSE: Gets the license creation date timestamp.
+
+    RESULT: License creation date timestamp.
+
+    RETURN CODES: ELAFailException, ELAProductIdException, ELATimeException, ELALicenseKeyException
+    ELATimeModifiedException
+*)
+
+function GetLicenseCreationDate: TDateTime;
+
+(*
+    FUNCTION: GetLicenseActivationDate()
+
+    PURPOSE: Gets the activation creation date timestamp.
+
+    RESULT: Activation creation date timestamp.
+
+    RETURN CODES: ELAFailException, ELAProductIdException, ELATimeException, ELALicenseKeyException
+    ELATimeModifiedException
+*)
+
+function GetLicenseActivationDate: TDateTime;
+
+(*
+    FUNCTION: GetLicenseMaintenanceExpiryDate()
+
+    PURPOSE: Gets the license maintenance expiry date timestamp.
+
+    RESULT: Maintenance expiry date timestamp.
+
+    RETURN CODES: ELAFailException, ELAProductIdException, ELATimeException, ELALicenseKeyException
+    ELATimeModifiedException
+*)
+
+function GetLicenseMaintenanceExpiryDate: TDateTime;
+
+(*
+    FUNCTION: GetLicenseMaxAllowedReleaseVersion()
+
+    PURPOSE: Gets the maximum allowed release version of the license.
+
+    RESULT: Max allowed release version.
+
+    EXCEPTIONS: ELAFailException, ELAProductIdException, ELATimeException,
+    ELATimeModifiedException, ELAProductVersionNotLinkedException,
+    ELABufferSizeException
+*)
+
+function GetLicenseMaxAllowedReleaseVersion: UnicodeString;
+
+(*
+    FUNCTION: GetLicenseOrganizationName()
+
+    PURPOSE: Gets the organization name associated with the license.
+
+    RESULT: Organization name.
+
+    EXCEPTIONS: ELAFailException, ELAProductIdException, ELATimeException,
+    ELATimeModifiedException, ELAProductVersionNotLinkedException,
+    ELABufferSizeException
+*)
+
+function GetLicenseOrganizationName: UnicodeString;
+
+(*
+    FUNCTION: GetActivationId()
+
+    PURPOSE: Gets the activation id.
+
+    RESULT: Activation id.
+
+    EXCEPTIONS: ELAFailException, ELAProductIdException, ELATimeException,
+    ELATimeModifiedException, ELABufferSizeException
+*)
+
+function GetActivationId: UnicodeString;
+
+(*
+    FUNCTION: GetActivationMode()
+
+    PURPOSE: Gets the mode of activation (online or offline).
+
+    RESULT: Activation mode.
+
+    EXCEPTIONS: ELAFailException, ELAProductIdException, ELALicenseKeyException,
+    ELATimeModifiedException, ELABufferSizeException
+*)
+
+function GetActivationMode: UnicodeString;
+
 (*
     FUNCTION: GetLicenseExpiryDate()
 
     PURPOSE: Gets the license expiry date timestamp.
 
     RESULT: License expiry date timestamp
 
-    EXCEPTIONS: ELAFailException, ELAProductIdException,
+    EXCEPTIONS: ELAFailException, ELAProductIdException, ELALicenseKeyException
```

Comment:
@muneebkq A value of 0 indicates it has no expiry i.e a lifetime license. Add this to the result of GetLicenseExpiryDate(). Also to the maintenance getter.